authorizations/authorization-ranger/src/main/resources/authorization-defs/authorization-hive.properties.template

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# If you enable these property configurations,
+# The RangerAuthorizationHivePlugin will use these properties to override the default configuration in the program.
+
+# Ranger Hive privileges enum RangerHivePrivilege: create, alter, drop, index, lock, select, insert, update, delete, read, write, all
+# defined in the `authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerPrivilege.java`
+# Format: gravitino.authorization.owner.privileges = <RANGER-PRIVILEGE>,<RANGER-PRIVILEGE>,...
+# gravitino.authorization.owner.privileges = all
+
+# Gravitino privileges defined in the `api/src/main/java/org/apache/gravitino/authorization/Privilege.java`
+# Format: authorization.privilege.mapping.<GRAVITINO-PRIVILEGE> = <RANGER-PRIVILEGE>,<RANGER-PRIVILEGE>,...
+# gravitino.authorization.privilege.mapping.CREATE_SCHEMA = create
+# gravitino.authorization.privilege.mapping.CREATE_TABLE = create
+# gravitino.authorization.privilege.mapping.SELECT_TABLE = read,select
```

Comment:
Do we handle the scenarios like having prefix and trailing whitespace in between policies for the conf value? like:

```
gravitino.authorization.privilege.mapping.CREATE_TABLE = create, aaa, xxx,   aaa
```